docs/docs/caching.md

```diff
@@ -51,7 +51,12 @@ When deploying with Now, follow the instructions in the [Now documentation](http
 
 ---
 
-<sup>1</sup> You can use 'no-cache' instead of 'max-age=0, must-revalidate'. Despite what the name might imply, 'no-cache' permits a cache to serve cached content as long as it validates the cache freshness first. <sup>[2][3] </sup> In either case, clients have to make a round trip to the origin server on each request. However, if you are correctly utilizing ETags or Last-Modified validation you will avoid downloading assets when the cached copy is still valid (e.g. the file hasn't changed on the origin server since it was cached).
+<sup>
```

Comment:
This one's kind of obnoxious... could we ignore prettier for this paragraph?

Reply:
Yeah. Sure. I'll update this.

Reply:
Hmm, I don't think I care that much. This whole page is weird: https://www.gatsbyjs.org/docs/caching/

1. There are no [2] or [3] references in the text
2. The footnotes don't have any links at all, making them annoying to reach.